Basic HTML fleshed out

yuppiecruncher opened the pull request:
As noted, I have written the HTML skeleton. Note I added some styling in the HTML to make the pages more viewable. These will be removed when styling begins.

yuppiecruncher committed:
added basic html and some formatting just to make it manageable.

yuppiecruncher committed:
Minor formatting changes for validation.

yuppiecruncher committed:
made css changes to forked adoptadog

yuppiecruncher commented:
Completed CSS styling as per instructions, following the mockup provided.

jimmynono commented on blog.html:
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
```
You'll want the meta tag referenced in week one on all of your HTML pages

yuppiecruncher replied:
I'm not sure which tag you are referring to. Do you mean `<meta name="viewport" content="width=device-width, initial-scale=1">`

Is it week one of HTML200? Was it referenced in a video? any help would be appreciated

jimmynono commented on blog.html:
```diff
@@ -0,0 +1,63 @@
+  <body>
+    <div class="header-img">
+      <a href="index.html">
+      <img class="logo" src="images/adoptadog-logo.svg" alt="Adopt-a-dog logo.">
```
This will need to be indented in one level from the anchor tag

yuppiecruncher replied:
Done

jimmynono commented on blog.html:
```diff
@@ -0,0 +1,63 @@
+      <a class="navlink" href="blog.html">Blog</a>
+    </div>
+
+    <div class="doggo-bloggo-container">
```
doggo bloggo? 

LOVE IT!

yuppiecruncher replied:
Gotta have fun with your work!

jimmynono commented on blog.html:
```diff
@@ -0,0 +1,63 @@
+    </div>
+
+    <footer>
+      <img src="images/adoptadog-logo.svg" style="width:500px" alt="Adopt-a-dog logo.">
```
You'll want to get rid of all your inline css and do it all from your css files

yuppiecruncher replied:
Done.

jimmynono commented on dogs.html:
```diff
@@ -0,0 +1,130 @@
+    <a class="navlink" href="blog.html">Blog</a>
+  </div>
+
+<div class="centering-container">
```
This would be a good use case for the main element

yuppiecruncher replied:
Did some homework on the <main> element. Very cool. Thanks for the tip!

jimmynono commented on dogs.html:
```diff
@@ -0,0 +1,130 @@
+      <div class="doggo-card">
+        <img class="card-img" src="images/murphy-card.jpg" alt="">
+        <div class="card-text">
+        <h4>Murphy</h4>
```
Anything nested in the dif needs to be indented in

yuppiecruncher replied:
Done.

jimmynono commented on dogs.html:
```diff
@@ -0,0 +1,130 @@
+      <h1>Here are all of our Dogs!</h1>
+
+      <div class="doggo-card">
+        <img class="card-img" src="images/murphy-card.jpg" alt="">
```
Don't forget to fill in  your alt tags

yuppiecruncher replied:
haha, the basics always get me. Thanks.

jimmynono commented on dogs.html:
```diff
@@ -0,0 +1,130 @@
+          <p><strong>Cost to Adopt: </strong>$123.45</p>
+          <p>Most angery pupper I have ever seen pupperino many pats fat boi, boof waggy wags. What a nice floof doggorino clouds blep he made many woofs woofer big ol pupper.</p>
+        </div>
+        <a class="adopt-btn" href="#">Adopt</a>
```
Tip, these will be buttons that do JS, not a navigation link

yuppiecruncher replied:
Good to know. I will replace this with a button.

jimmynono commented on styles/style.css:
```diff
@@ -0,0 +1,219 @@
+  margin-top: 1em;
+}
+
+.tagline {
```
watch your css formatting to make sure things are aligned right.
```
.tagline {
  color: black;
}
```

yuppiecruncher replied:
I'm not sure what formatting you mean. I removed an extra space on line 19.

jimmynono commented on styles/style.css:
```diff
@@ -0,0 +1,219 @@
+
+.tagline {
+  color: 	#81B642;
+  font-size: 2vw;
```
I'd prefer you to keep to REMS or EMS for font sizes, it's a bit more standard at this point. If you really want to stick to vw thats's fine, but I'd encourage you to get used to REMS and EMS

yuppiecruncher replied:
Is there a way to get the text to behave like vw using rems? I preferredd the vw method because it scales very nicely with the image when resizing windows. When using rems or ems, it always seems awkward.

jimmynono commented on styles/style.css:
```diff
@@ -0,0 +1,219 @@
+  background-color: #81B642;
+  border: none;
+  color: white;
+  padding: 5px 18px;
```
Keep to REMS/EMS for this, you'll want to be consistent

yuppiecruncher replied:
Done.

jimmynono commented on styles/style.css:
```diff
@@ -0,0 +1,219 @@
+  width: 80%;
+}
+
+.blogimg {
```
when naming classes with two words it's best to kabob case them
```
.blog-img {
  float: left;
}
```

yuppiecruncher replied:
Done.

yuppiecruncher committed:
Commit for testing repo

yuppiecruncher committed:
Made units consistent and responsive. Replaced all px with em units. Left VW unit for tagline because it works well and looks good at all window sizes.

yuppiecruncher committed:
removed styling references from HTML -> added to styles.css. Made changes recommended on pull request. Will work on removing floats this week. Added meta tag options as per first video lecture.

yuppiecruncher committed:
Incorporated JS into index.html and dogs.html created js directory and file for reference and management.

yuppiecruncher committed:
Added breed information as per my re-read of the instructions.

yuppiecruncher committed:
Made small changes to css and html in order to bring it into mobile-first design.

yuppiecruncher committed:
Continued work on responsive grid. still need to add header, footer and side grid.

yuppiecruncher committed:
responsive blog preview complete.

yuppiecruncher committed:
further refining styles. fixed footer issue on dogs.html.

yuppiecruncher committed:
finished responsive dog tiles on dogs.html

yuppiecruncher committed:
Check for errors complete.

yuppiecruncher committed:
refined backgroun-image property to include gradient in case of missing image file. Photo position changed to show the dogs face as per the mockup. All images appear to be sized appropriately for responsive design.

yuppiecruncher committed:
margin edit on .blog-prev-grid

yuppiecruncher committed:
minor edits to headings

yuppiecruncher committed:
added main.js, connected and created blog-post scripts.

added for loop to iterate over posts array.

Added alert for form submission on checkout form

Added form data collection into an array object. Would like to turn this 
into a key:value pair system.

yuppiecruncher committed:
added jquery and refactored scripts for checkout form and cart-total

yuppiecruncher committed:
Added alert to checkout page after refactoring to jquery.

yuppiecruncher committed:
added some alerts to the cart script. Added the dog tile animation to the dogs page.

yuppiecruncher committed:
Create README.md

yuppiecruncher committed:
Update README.md

yuppiecruncher committed:
Update README.md

yuppiecruncher committed:
Update README.md

yuppiecruncher commented:
Added README.md

yuppiecruncher committed:
Added testing section for documentation exercise.

yuppiecruncher committed:
Merge branch 'master' of https://github.com/yuppiecruncher/adopt-a-dog-spring2019

yuppiecruncher committed:
Update README.md

yuppiecruncher committed:
Edited testing.txt

yuppiecruncher committed:
Merge branch 'master' of https://github.com/yuppiecruncher/adopt-a-dog-spring2019

jimmynono commented on README.md:
```diff
@@ -0,0 +1,31 @@
+# Adopt-a-dog Website
```
Nice job with the formatting!
